setup.py

```diff
@@ -86,15 +86,34 @@ def is_platform_mac():
     if not _CYTHON_INSTALLED:
         raise ImportError('No supported version of Cython installed.')
     from Cython.Distutils import build_ext as _build_ext
+    from Cython import Tempita as tempita
```

Comment:
Found the related discussion in cython-users. The same as Scipy, added fallback logic from Cython, standard Tempita, otherwise raise. Not includ Tempita as dependency because we can expect to use it from Cython.

Another option is always use standard Tempita (and add it to dependency).

https://groups.google.com/forum/#!topic/cython-users/LViuQUs4iQo